include/support.h

```diff
@@ -33,6 +33,8 @@
 
 #include <SDL.h>
 
+#define PI 3.141592653589793238462643383279f
```

Comment:
@dreamer  - maybe I should drop the 'f', because this is probably being truncated down a `float`, correct?